lib/src/booleans/boolean_disjoint.dart

```diff
@@ -0,0 +1,365 @@
+import '../../helpers.dart';
+import '../../meta.dart';
+import '../line_intersect.dart';
+import '../polygon_to_line.dart';
+import 'boolean_point_in_polygon.dart';
+
+/**
+ * Boolean-disjoint returns (TRUE) if the intersection of the two geometries is an empty set.
+ *
+ * @name booleanDisjoint
+ * @param {Geometry|Feature<any>} feature1 GeoJSON Feature or Geometry
+ * @param {Geometry|Feature<any>} feature2 GeoJSON Feature or Geometry
+ * @returns {boolean} true/false
+ * @example
+ * var point = turf.point([2, 2]);
+ * var line = turf.lineString([[1, 1], [1, 2], [1, 3], [1, 4]]);
+ *
+ * turf.booleanDisjoint(line, point);
+ * //=true
+ */
+bool booleanDisjoint(GeoJSONObject feature1, GeoJSONObject feature2) {
+  var bool = true;
+  flattenEach(feature1, (flatten1, featureIndex, multiFeatureIndex) {
+    flattenEach(feature2, (flatten2, featureIndex, multiFeatureIndex) {
+      if (bool == false) {
+        return false;
```

Comment:
what do you want to do with this return statement? should it return for the whole funciton `booleanDisjoint`?

Reply:
I am still working on this. Not finished.